cli/test/lib/cypress_spec.js

```diff
@@ -105,6 +105,7 @@ describe('cypress', function () {
       .then((args) => {
         expect(args.spec).to.equal('foo')
         expect(args.autoCancelAfterFailures).to.equal(4)
+        expect(args.runnerUi).to.beUndefined
```

Comment:
Is this a valid matcher? I thought for chai, it was `to.be.undefined`

Reply:
updated to use `to.be.undefined` [388ef5c](https://github.com/cypress-io/cypress/pull/27582/commits/388ef5cb5ddb53b019ec7fc818b23c61ab4b243e)